src/services/llm_router.py

```
@@ -0,0 +1,171 @@
+            return GRANITE_ENDPOINT
+
+        # Try config manager first (dynamic provider config)
+        try:
+            from config.settings import config_manager
+            from utils.container_utils import transform_localhost_url
+
+            cfg = config_manager.get_config()
+            if cfg and cfg.providers.ollama.endpoint:
+                return transform_localhost_url(cfg.providers.ollama.endpoint)
+        except Exception:
+            pass
```

Ruff (0.15.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)